spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/StreamDefinitionController.java

```diff
@@ -16,10 +16,19 @@
 
 package org.springframework.cloud.dataflow.server.controller;
 
+import java.util.ArrayList;
```

Comment:
Actually - I'm sorry, and nevermind my comment. What you have in your PR is correct (*java.* should precede *org.*)... so apparently this is a case of someone else having different settings previously ;)